src/Auth0.AuthenticationApi.Client/Builders/WsFedUrlBuilder.cs

```diff
@@ -0,0 +1,34 @@
+﻿using JetBrains.Annotations;
+
+namespace Auth0.AuthenticationApi.Client.Builders
+{
+    public class WsFedUrlBuilder : UrlBuilderBase<WsFedUrlBuilder>
```

Comment:
I would add an additional method here `WithWctx` or `WithState` that sets the `wctx` to a value (this is the state).

Ideally this should be a string or a key value pair (that is then flattened to a string, like `xcrf=abc&ru=/foo` (make sure you also add url encoding here)